### DISCUSSION TITLE
Make "FastLane" a top level header

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#221] [Ideas]
Now that I know what I'm looking for - FastLane setup.   I notice why I never really saw this in the Docs...
It is hidden below Continuous Deployment.   I think if Fastlane is your choice for CD... it should get a bit better billing in the Overview Table of Contents.
![Screenshot 2024-09-05 at 1 05 30 PM](https://github.com/user-attachments/assets/c12ca2cb-66b7-4888-8eae-8c406a74741a)

**Comment by davidakoontz** [comment]
oh - and correct the spelling  fastlaene - dosn't work :)

**Comment by aabewhite** [comment]
Thanks - done!
